src/atomate2/forcefields/jobs.py

```diff
@@ -42,19 +86,25 @@ class ForceFieldRelaxMaker(Maker):
         Keyword arguments that will get passed to :obj:`Relaxer.relax`.
     optimizer_kwargs : dict
         Keyword arguments that will get passed to :obj:`Relaxer()`.
+    calculator_args : Sequence[Any]
+        Keyword arguments that will get passed to the ASE calculator.
+    calculator_kwargs : dict
+        Keyword arguments that will get passed to the ASE calculator.
     task_document_kwargs : dict
         Additional keyword args passed to :obj:`.ForceFieldTaskDocument()`.
     """
 
     name: str = "Force field relax"
-    force_field_name: str = "Force field"
+    force_field_name: str = f"{MLFF.Forcefield}"
     relax_cell: bool = True
     steps: int = 500
     relax_kwargs: dict = field(default_factory=dict)
     optimizer_kwargs: dict = field(default_factory=dict)
+    calculator_args: Sequence[Any] = field(default_factory=list)
```

Comment:
given every arg can be passed as a kwarg, i would get of `calculator_args`. args are more likely to cause footguns if users get the order of args wrong or when a breaking change in upstream code changes the order. so there are significant safety benefits from forcing users to pass everything as kwargs.

Reply:
Removed the args from the forcefield jobs/md jobs and calculator. I added it yesterday for Nequip, but I agree with you that it can cause unexpected problems